docs/functions-testing-tools/function-stubs.md

```diff
@@ -141,6 +141,40 @@ Only for functions that both translate and escape \(`esc_html__()`, `esc_html_x(
 
 Please note how `Functions\stubTranslationFunctions()` creates stubs for functions that _echo_ translated text, something not easily doable with `Functions\stubs()` alone.
 
+### Pre-defined stubs for WP URL functions
+
+One very common and repetitive task testing WordPress code without WordPress loaded is to stub
+WordPress URLs, a task made easy by the function:
+
+**`Functions\stubWpUrlFunctions()`**
+
+When called, it will create a stub for _all_ the following functions:
+
+* `home_url()`
+* `get_home_url()`
+* `site_url()`
+* `get_site_url()`
+* `admin_url()`
+* `get_admin_url()`
+* `content_url()`
+* `rest_url()`
+* `get_rest_url()`
+* `includes_url()`
+* `network_home_url()`
+* `network_site_url()`
+* `network_admin_url()`
+* `user_admin_url()`
+* `wp_login_url()`
+
+The function accepts as first argument the domain to use for subbing the URLs, default to `example.org`.
+
+E.g., after `Functions\stubWpUrlFunctions()`, the `home_url()` function returns `https://example.org`,
+but after `Functions\stubWpUrlFunctions('acme.com')`, the `home_url()` function returns `https://acme.org`.
```

Comment:
```suggestion
E.g., calling `Functions\stubWpUrlFunctions()`, the `home_url()` function returns `https://example.org`,
but calling `Functions\stubWpUrlFunctions('acme.com')`, the `home_url()` function returns `https://acme.org`.
```

The word "after" could give the impression that some form of caching/remembering of previously passed function call arguments is being done. This small change removes that confusion.